### PR TITLE
Debian packaging

### PR DESCRIPTION
**Pull request by digizeph**
- add debian packaging folder
- add bsd license
- update author info

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -7,8 +7,8 @@
     version='0.2',
     description='High-level file IO library',
     url='https://github.com/CAIDA/pywandio',
-    author='Alistair King, Chiara Orsini',
-    author_email='bgpstream-info@caida.org',
+    author='Alistair King, Mingwei Zhang',
```
Comment (alistairking) [inline setup.py:10]: why did you remove chiara's name from the author list?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,22 @@
+Copyright (C) 2019 The Regents of the University of California.
```
Comment (alistairking) [inline LICENSE:1]: i think these should be 2018 -- probably earlier but unfortunately i didn't save the commit history when i ported the code from data concierge.

Comment (alistairking) [reply LICENSE:1]: https://github.com/CAIDA/pywandio/commit/8ebc34828477e57dcdebb2d66cb0cba77fedc9fe

File: setup.py (hunk 1 above)
Comment (digizeph) [reply setup.py]: Sorry, my mistake! I only looked at git commit authors. Added her back in f255036

File: LICENSE (hunk 2 above)
Comment (digizeph) [reply LICENSE]: Updated `LICENSE` and `debian/copyright` dates in e7c795b
